<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThongKe</class>
 <widget class="QMainWindow" name="ThongKe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Bahnschrift</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding-top: 10px;</string>
        </property>
        <property name="text">
         <string>THỐNG KÊ HÔM NAY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton{
border:none;
	background-color: rgb(255, 170, 127);
}
QWidget{
background-color: rgb(255, 170, 127);
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Truyện</string>
              </property>
              <property name="icon">
               <iconset resource="new_resource.qrc">
                <normaloff>:/newPrefix/icons/books.png</normaloff>:/newPrefix/icons/books.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton{
border:none;
	background-color: rgb(255, 170, 127);
}
QWidget{
background-color: rgb(85, 170, 255);
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Hóa đơn bán </string>
              </property>
              <property name="icon">
               <iconset resource="new_resource.qrc">
                <normaloff>:/newPrefix/icons/bill (3).png</normaloff>:/newPrefix/icons/bill (3).png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_10" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton{
border:none;
	
}
QWidget{
background-color: rgb(255, 234, 110);
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Hóa đơn nhập</string>
              </property>
              <property name="icon">
               <iconset resource="new_resource.qrc">
                <normaloff>:/newPrefix/icons/invoice.png</normaloff>:/newPrefix/icons/invoice.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_11" native="true">
           <property name="styleSheet">
            <string notr="true">QPushButton{
border:none;
	
}
QWidget{
background-color: rgb(43, 190, 17);
}
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Thuê truyện</string>
              </property>
              <property name="icon">
               <iconset resource="new_resource.qrc">
                <normaloff>:/newPrefix/icons/reading-book.png</normaloff>:/newPrefix/icons/reading-book.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <family>Bahnschrift</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="dothi_2" native="true">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="new_resource.qrc"/>
 </resources>
 <connections/>
</ui>
